protocols/v2/binary-sv2/no-serde-sv2/codec/src/lib.rs

```diff
@@ -120,7 +120,7 @@ impl<'a> From<Vec<u8>> for EncodableField<'a> {
 }
```

Comment:
There is no need to change this file. This file should remain exactly equal to the one that is the main branch.